packages/react-intersection-observer/README.md

```diff
@@ -35,15 +59,15 @@ This component also allows you to customize the rendered markup. You can pass `c
 
 ### Lifecycle
 
-You may change any prop on an `IntersectionObserver` component, and the component will do the minimum amount of work to unobserve/ re-observe with the new fields. The most expensive updates to make are changing `threshold`, `root`, `rootMargin`, and `wrapperComponent`, as these require disconnecting the old observer and recreating a new one.
+You may change any option on a `useIntersection` hook, or any prop on an `IntersectionObserver` component, and the minimum amount of work will be performed to unobserve/ re-observe with the new fields. The most expensive updates to make are changing `threshold`, `root`, `rootMargin`, and, in the case of the component version, `wrapperComponent`, as these require disconnecting the old observer and recreating a new one.
```

Comment:
IMO `You may change` feels weird, I think it would flow better to describe this in terms of `When useIntersection is passed new values or IntersectionObserver receives new props`